healthcheck/healthcheck.go

```diff
@@ -24,10 +24,10 @@ func Poll() error {
 	if client == nil {
 		return fmt.Errorf("Can not create RancherClient, No credentials found")
 	}
-
+	c, _ := lru.New(500)
```

Comment:
500 as thats the max number of containers that Rancher supports per host (as per Will's words)